lib/widgets/homepage.dart

```diff
@@ -18,6 +18,7 @@ class Home extends StatefulWidget {
 class _HomeState extends State<Home> {
   static const List<Widget> _appBars = [
     FriendsListAppBar(),
+    // Disabled due to DownloadManager not working on desktop
```

Comment:
This comment is not accurate anymore right?

Reply:
that is correct, yes